lighthouse-core/gather/driver.js

```diff
@@ -472,16 +475,17 @@ class Driver {
    * @param {!Object} options
    * @return {!Promise}
    */
-  gotoURL(url, options) {
-    const _options = options || {};
-    const waitForLoad = _options.waitForLoad || false;
-    const disableJS = _options.disableJavaScript || false;
-    const pauseAfterLoadMs = (_options.flags && _options.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
+  gotoURL(url, options = {}) {
+    const waitForLoad = options.waitForLoad || false;
+    const disableJS = options.disableJavaScript || false;
+    const pauseAfterLoadMs = (options.flags && options.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
+    const maxWaitMs = (options.flags && options.flags.maxWaitForLoad) ||
```

Comment:
I would say that would lead to a useless lighthouse run that wouldn't return any results so filling in default is fine with me :)